app/views/visitor/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>All Teams in the playoff race</h1>
+<h1>* only includes the teams currently in the playoffs*</h1>
+
+
+<%# if current_user %>
```

Comment:
I like this conditional. I think you should keep it and have the feature to show the correct log in log out based on the users current session